test/api/buffer/api_test.go

```diff
@@ -193,15 +194,20 @@ func RunApiServer(t *testing.T, binary string, storageApiHost string) (apiUrl st
 	args := []string{fmt.Sprintf("--http-port=%d", port)}
 
```

Comment:
etcd timeout je `30s`, test timeout by mal byt vacsi, napr. `45s`.

Reply:
Teď nevím, jestli jsem tohle správně pochopil. Ten client v `Cleanup` by měl mít `45s`?

Reply:
Nie (len to nebolo sucastou tohto PR, tak som to okomentoval, na prazdnom riadku).

Je to definovane tu (mozes z toho spravit konstantu na zaciatku suboru, ak chces):
https://github.com/keboola/keboola-as-code/blob/b0c0a9d4fa0d7fa017f514d0e2379b7a512e1d9e/test/api/buffer/api_test.go#L153

Reply:
A upravit aj chybovu hlasku, pridat `%s`: https://github.com/keboola/keboola-as-code/blob/b0c0a9d4fa0d7fa017f514d0e2379b7a512e1d9e/test/api/buffer/api_test.go#L160